utils/utils.py

```
@@ -325,4 +325,8 @@ def ensure_podman_running(client=None):
         else:
             print("Podman machine started successfully.")
     else:
-        print("Podman is already running.")
\ No newline at end of file
+        print("Podman is already running.")
+
+def normalise_url(version, url):
+    stdout, stderr = run_command(f"opm alpha list bundles {url} | grep {version} | awk '{{print $6}}' | sed 's/registry.redhat.io/registry.stage.redhat.io/'")
```

Ruff (0.15.0)

[warning] Unpacked variable `stderr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)